### PR TITLE
add sqlite3 through sequelize

### PR DESCRIPTION
**Pull request by dcepil**

### CODE REVIEW

**Comment by VuongRon** [comment]
[SQLite or other migration](https://trello.com/c/xz0WJF9b/17-sqlite-or-other-migration)

File: migrations/20201115230456-create-user.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      discordId: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
```
Comment (VuongRon) [inline migrations/20201115230456-create-user.js:14]: Just FYI you don't HAVE to have `id` as PK. If you want to rely on `discordId` I think you can just set the discord ID block with `primaryKey: true`.

**Review by VuongRon — COMMENTED** [review]
image

Pog. Just look at the 1 comment, if it's intentional then its approved.

File: migrations/20201115230456-create-user.js (hunk 1 above)
Comment (dcepil) [reply migrations/20201115230456-create-user.js:14]: This is intentional, I don't want to use ```discordId``` as PK.

**Pull request merged**